src/main/java/de/tum/in/www1/artemis/repository/metrics/ExerciseMetricsRepository.java

```diff
@@ -80,7 +80,7 @@ CASE WHEN TYPE(e) = ProgrammingExercise THEN TREAT(e AS ProgrammingExercise).all
      * @return the score for the user in each exercise
      */
     @Query("""
-            SELECT new de.tum.in.www1.artemis.web.rest.dto.metrics.ScoreDTO(s.exercise.id, CAST(COALESCE(s.lastRatedScore, s.lastScore, 0) AS DOUBLE))
+            SELECT new de.tum.in.www1.artemis.web.rest.dto.metrics.ScoreDTO(s.exercise.id, CAST(COALESCE(s.lastRatedScore, 0) AS DOUBLE))
```

Comment:
Why do you no longer send the lastScore here as fallback value?

Reply:
I wonder about this as well

Reply:
FD: This should not take the practice score as a fallback for the "Exercise Performance" overview, I think only the rated scores should be taken into account there.

Reply:
Yeah, previously we did not consider practice exercises at all, or at least by accident. For the performance overview we would only like to show rated scores and not practice scores. I just verified that it does it correctly now and fixed also the average.